{{ cookiecutter.folder_name }}/{{ cookiecutter.import_name }}/tests/test_blank.py

```diff
@@ -1,11 +1,12 @@
 """
-This document and it's single test are included only so that first-time builds
+This document and its single test are included only so that first-time builds
 with doctr will succeed and create an empty document.
 """
-import pytest
 import logging
+
+
 logger = logging.getLogger(__name__)
 
-def test_blank():
-    1/0
 
+def test_blank():
+    raise ZeroDivisionError()
```

Comment:
My one and only critique. This is completely subjective. Given that `assert false` is probably the most generic failure type for use in pytest, it's probably also the best example for a starter test. This is minor enough that I'm approving the PR regardless.